config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -163,7 +163,7 @@ susy.quest.db.51.desc=§6LV Electric Motors §rare one of the most important com
 susy.quest.db.52.title=LV Electric Piston
 susy.quest.db.52.desc=The §6LV Electric Piston§r is also another important component of electric machines, but it will be required to make the Steam Macerator and Forge Hammer.
 susy.quest.db.53.title=Cathodes and Anodes
-susy.quest.db.53.desc=Combining a plate of §6Lead §rwith some wiring creates §6Lead Anodes and Cathodes§r. If they are combined with §6Sulfuric Acid§r, they can make a Lead Acid Battery.
+susy.quest.db.53.desc=Combining a plate of §5Lead§r with some wiring creates §5Lead Anode§r. By covering it in §5Lead Oxide Paste§r, you can get the §5Lead Cathode§r. If they are combined with §6Diluted Sulfuric Acid§r, they can make a Lead Acid Battery.
```

Comment:
`creates §5Lead Anode§r` -> `creates a §5Lead Anode§r`
`By covering it in §5Lead Oxide Paste§r, you can get the §5Lead Cathode§r` -> `By covering an anode in §5Lead Oxide Paste§r, you can also create a §5Lead Cathode§r`

Reply:
that's outdated code